compose/config/tcp/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 3
```

Comment:
does this file need to be in a `tcp` subdir? it seems like it shouldn't need to be